File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.ArrayList;
import java.util.Arrays;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        ArrayList<Integer> arr = new ArrayList<>();

        String[] strs = br.readLine().split(" ");
        for (int i = 0; i < n; i++) {
            arr.add(Integer.parseInt(strs[i]));
        }
        int wantNum = Integer.parseInt(br.readLine());
        int ans = 0;
        for (int i = 0; i < n; i++) {
            if(arr.get(i) == wantNum) ans++;
        }
        System.out.println(ans);
    }
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.py
import sys
inp = sys.stdin.readline

n = int(inp())
arr = [0] * 201 # -100 ~ 100 (0 포함)

nums = map(int, inp().split())

for num in nums:
    arr[num+100]+=1

v=int(inp())
print(arr[v + 100])